Gems/GraphCanvas/Code/CMakeLists.txt

```diff
@@ -78,3 +80,35 @@ if (PAL_TRAIT_BUILD_HOST_TOOLS)
 
 
 endif ()
+
+################################################################################
+# Tests
+################################################################################
+if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
+
+    ly_add_target(
+        NAME GraphCanvas.Tests ${PAL_TRAIT_TEST_TARGET_TYPE}
+        NAMESPACE Gem
+        FILES_CMAKE
+            graphcanvas_tests_files.cmake
+        COMPILE_DEFINITIONS
+            PRIVATE
+                GRAPHCANVAS_TESTS
+        INCLUDE_DIRECTORIES
+            PRIVATE
+                Tests
+                Source
+        BUILD_DEPENDENCIES
+            PRIVATE
+                AZ::AzTest
+                Gem::GraphCanvas
```

Comment:
Seems like you only need to include Gem::GraphCanvasWidgets as build dependency, so what is Gem::GraphCanvas used for? and I don't think you need runtime dependency for your unit test (which is used to bring and copy shared libs)?